coderd/coderd.go

```diff
@@ -175,6 +175,10 @@ func New(options *Options) *API {
 	if options.AgentInactiveDisconnectTimeout == 0 {
 		// Multiply the update by two to allow for some lag-time.
 		options.AgentInactiveDisconnectTimeout = options.AgentConnectionUpdateFrequency * 2
+		// Set a minimum timeout to avoid disconnecting too soon.
+		if options.AgentInactiveDisconnectTimeout < 2*time.Second {
```

Comment:
> 2*time.Second

Interesting, doesn't `make fmt` complain about it?

Reply:
This is actually fmt's handiwork. It groups operands when there are multiple operators.

Reply:
TIL!